js/index.js

```diff
@@ -1,7 +1,47 @@
 // Iteration 1: Names and Input
+const hacker1 = 'Nidian'
+console.log(`"The driver's name is ${hacker1}"`)
 
+const hacker2 = 'Maria'
+console.log(`"The navigator's name is ${hacker2}"`)
 
 // Iteration 2: Conditionals
 
+if (hacker1Long > hacker2Long) {
+  console.log(`The driver has the longest name, it has ${hacker1Long } characters`)
+} else if (hacker2Long > hacker1Long) {
+  console.log(`It seems that the navigator has the longest name, it has 
+  ${hacker2Long} characters`)
+} else {
+  console.log(`Wow, you both have equally long names, ${hacker2Long} characters!`)
+}
 
-// Iteration 3: Loops
\ No newline at end of file
+// Iteration 3: Loops
+// Ejercicio 3.1
+let driver = ''
+for (let i = 0; i < hacker1.length; i++ ) {
+  driver += hacker1[i].toUpperCase() + ' '
+}
+console.log(driver);
+
+//Ejercicio 3.2
+let navigator = ''
+for (let i = hacker2.length -1; i >= 0 ; i--) {
```

Comment:
Un par de cosas sobre la indentación ;)

for (let i = hacker2.length "**add white space here**"; i >= 0"**remove white space here**"; i--)